core/src/components/radio-group/test/basic/radio-group.e2e.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from '@playwright/test';
+import type { Locator } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+import type { E2EPage } from '@utils/test/playwright';
+
+test.describe('radio-group: basic', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/radio-group/test/basic`);
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot()).toMatchSnapshot(`radio-group-diff-${page.getSnapshotSettings()}.png`);
```

Comment:
Do we need to screenshot the entire page, or can we screenshot just `ion-radio-group`?